qcodes/tests/dataset/helper_functions.py

```diff
@@ -0,0 +1,33 @@
+from typing import Sequence, Tuple, Dict
+
+import numpy as np
+
+
+def verify_data_dict(data: Dict[str, Dict[str, np.ndarray]],
```

Comment:
I might be missing a point here but should this function do more checks? like if names of the `expected_names[param]` are exactly the keys of the inner dicts?